vendor-extra/JatsContentBundle/src/Handler/JatsContentHandler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\Handler;
+
+use FluentDOM\DOM\Attribute;
+use FluentDOM\DOM\Document;
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Xpath;
+use Libero\ContentPageBundle\Handler\ContentHandler;
+use UnexpectedValueException;
+use function array_merge;
+use function Libero\ContentPageBundle\text_direction;
+
+final class JatsContentHandler implements ContentHandler
```

Comment:
the name `JatsContentHandler` (and perhaps `JatsContentBundle`) is misleading if, as far as JATS variations go, this can support eLife-like JATS or Libero-constrained JATS rather than the totality of JATS content. This makes a conversion step still necessary from non-Libero-constrained JATS to Libero-constrained JATS, _unless_ this frontend and all the various services make efforts to support various options.

Reply:
On the ingestion validation side of things, I guess a Schema(-tron?) could be written to validate it's Libero-constrained JATS, don't know how difficult it is to do

Reply:
Indeed, not yet clear _what_ JATS will be supported yet. I suspect it's going to be somewhere between DAR and JATS4R. There might be a certain amount of configuration too, so you could tweak selectors.

Reply:
It's not. You probably could use XSLT to do the same (presumably producing JSON which is a valid input for the patterns), but I suspect there's things not possible there (eg [having to replace placeholders in code afterwards](https://github.com/elifesciences/elife-ingest-xsl/blob/bd0e114f26af8a9fb506ac3ee6bb41d28d54d3fa/lib/xsl/jats-to-html.xsl#L1467-L1468)).